src/redis-benchmark.c

```diff
@@ -628,6 +628,50 @@ static void readHandler(aeEventLoop *el, int fd, void *privdata, int mask) {
     }
 }
 
+/* control the QPS of the thread of current client 
+*/
+static int controlQps(void *privdata) {
```

Comment:
if we are passing it the client anyway, why can't this function just get a pointer to client?

Reply:
> you mean the type of parameter of _controlQps()_?

yes (void *privdata)

Reply:
got it